docs/release-notes/2.22.0.md

```diff
@@ -0,0 +1,44 @@
+# Percona Monitoring and Management 2.22.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">September 21, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+- [PMM-8396](https://jira.percona.com/browse/PMM-8396): Backup Management: Retry failed scheduled backups
+- [PMM-8079](https://jira.percona.com/browse/PMM-8079): DBaaS: Upgrade Operators inside Kubernetes cluster
+- [PMM-7924](https://jira.percona.com/browse/PMM-7924): MySQL Performance Details dashboard: Add "Performance Schema Status Monitoring" chart
```

Comment:
Some  Additional text needed to get better explanation what the panel is presenting

Reply:
It seems self explanatory. If someone knows they need this, it's now available. If they don't know they need it, it needs to be explained, but this is not the right place to do that. It should be in the user guide.